otherlibs/dune-rpc-lwt/test/dune_rpc_lwt_tests.ml

```diff
@@ -77,7 +72,7 @@ let%expect_test "run and connect" =
   Lwt_main.run
     (let* root_dir = Lwt_io.create_temp_dir () in
      Sys.chdir root_dir;
-     let build = build_watch ~root_dir ~suppress_stderr:false in
+     let build = build_watch ~root_dir in
```

Comment:
There is no PR description but from the title I suppose the intent here is only to change the implementation. It seems strange, therefore, that here we're dropping the `suppress_stderr` argument.

Reply:
Sounds good.